docs/guides/webpack.md

```diff
@@ -20,44 +19,34 @@ react/lib/ReactContext
 react/lib/ExecutionEnvironment
 ```
 
-Depending on if you are using Webpack 1 or Webpack 2 you will need different configurations.
-
-### Webpack 1
+You will need the following configuration:
 
 ```js
 /* webpack.config.js */
 // ...
-externals: {
-  'cheerio': 'window',
-  'react/addons': true,
-  'react/lib/ExecutionEnvironment': true,
-  'react/lib/ReactContext': true
-}
+plugins: [
+  new webpack.IgnorePlugin(/react\/addons/),
+  new webpack.IgnorePlugin(/react\/lib\/ExecutionEnvironment/),
+  new webpack.IgnorePlugin(/react\/lib\/ReactContext/)
+]
 // ...
 ```
 
-### Webpack 2
-
-```js
-externals: {
-  'cheerio': 'window',
-  'react/addons': 'react',
-  'react/lib/ExecutionEnvironment': 'react',
-  'react/lib/ReactContext': 'react',
-},
-```
-
 
 ## React 0.14 Compatibility
 
 If you are using React 0.14, the instructions above will be the same but with a different list of
 externals:
 
-```
-cheerio
-react-dom
-react-dom/server
-react-addons-test-utils
+```js
+/* webpack.config.js */
+// ...
+plugins: [
+  new webpack.IgnorePlugin(/cheerio/),
+  new webpack.IgnorePlugin(/react-dom/),
```

Comment:
I believe this list of ignore plugins is actually for react 13 not 14, they're swapped for some reason?